config/payment_type.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+return [
+    '1' => 'cash_payment',
```

Comment:
```suggestion
    'cash_payment' => 1,
```
cái này phải như này chứ nhỉ? em đang dùng ntn vậy?

Reply:
dạ hơi ngược, tại bên form input e cho value=1, em đã sửa ạ :v